### PR TITLE
feat: InformationModal include working button

### PR DESCRIPTION
**Pull request by vicchirino**
# Objective
Include working button props on `InformationModal`

Add `WorkingActionProps` on `InformationModal` to allow primary button to display loading state when some action is happening. This is needed on `performance-ui` when we share performance reviews using this modal.

Add story to reflect this change
 
https://user-images.githubusercontent.com/3228237/110971716-f9aa6380-8339-11eb-8b6c-49add712efb1.mov

Include update on contentHeader layout to have same padding that the main content.

**Before:**
image

**After: **
image

### CODE REVIEW

**Comment by dougmacknz** [comment]
Hi @vicchirino, we have a pattern for `working` primary buttons on modals that was created in https://github.com/cultureamp/kaizen-design-system/pull/941. (The only reason I didn't do it for `InformationModal` there is because I didn't think this modal was used for submitting/confirming things, I was wrong.)

The usage looks like this:
`<ConfirmationModal confirmWorking={{ label: "Working…" }} />`
`<InputEditModal submitWorking={{ label: "Submitting…" }} />`

They can also receive an optional `labelHidden` prop like so (hence the object):
`<ConfirmationModal confirmWorking={{ label: "Working…", labelHidden: true }} />`

This PR would introduce a new pattern looking like this, which is going to be inconsistent:
`<InformationModal isWorking={true} workingLabel="Sharing review" />`

Can I please get you to look over that PR I linked above and copy the way it was done in `ConfirmationModal`. You will want to use the wording `confirmWorking` because the `InformationModal` has a `confirm` button (rather than a `submit` button).

The other behaviour missing here is that when `working` is enabled, `onDismiss` should be hijacked to block users from closing the modal while it's working. Again, please just copy the way it was done in https://github.com/cultureamp/kaizen-design-system/pull/941.

**Comment by keirah** [comment]
> Hi @vicchirino, we have a pattern for `working` primary buttons on modals that was created in #941. (The only reason I didn't do it for `InformationModal` there is because I didn't think this modal was used for submitting/confirming things, I was wrong.)
> 
> The usage looks like this:
> `<ConfirmationModal confirmWorking={{ label: "Working…" }} />`
> `<InputEditModal submitWorking={{ label: "Submitting…" }} />`
> 
> They can also receive an optional `labelHidden` prop like so (hence the object):
> `<ConfirmationModal confirmWorking={{ label: "Working…", labelHidden: true }} />`
> 
> This PR would introduce a new pattern looking like this, which is going to be inconsistent:
> `<InformationModal isWorking={true} workingLabel="Sharing review" />`
> 
> Can I please get you to look over that PR I linked above and copy the way it was done in `ConfirmationModal`. You will want to use the wording `confirmWorking` because the `InformationModal` has a `confirm` button (rather than a `submit` button).
> 
> The other behaviour missing here is that when `working` is enabled, `onDismiss` should be hijacked to block users from closing the modal while it's working. Again, please just copy the way it was done in #941.

Thanks for the detailed feedback @dougmacknz! @vicchirino is on a well deserved holiday right now so I'm going to try and move these changes forward.

**Comment by keirah** [comment]
Have pushed some changes, but would like to get them working with the [corresponding PR](https://github.com/cultureamp/performance-ui/pull/2768) that Vic has in the Performance-UI repo (to make sure it still works for his use case), before requesting a re-review.

File: draft-packages/modal/KaizenDraft/Modal/Presets/InformationModal.tsx [hunk 1]
```diff
@@ -34,75 +36,97 @@ type InformationModal = React.FunctionComponent<InformationModalProps>
...
+        workingLabelHidden: confirmWorking.labelHidden,
+      }
+    : {}
+
+  if (onConfirm) {
+    const confirmAction = { label: confirmLabel, onClick: onConfirm }
+
+    footerActions.push({ ...confirmAction, ...workingProps })
+  }
+
```
Comment (keirah) [inline draft-packages/modal/KaizenDraft/Modal/Presets/InformationModal.tsx:66]: The `ConfirmationModal` has a separate footerActions included

`  footerActions.push({
    label: dismissLabel,
    onClick: onDismiss,
    disabled: !!confirmWorking,
  })
`

Unsure if this is something that is required on the InformationModal?

**Review by dougmacknz — APPROVED** [review]

**Pull request merged**